priv/repo/migrations/20170421092531_create_course.exs

```diff
@@ -0,0 +1,17 @@
+defmodule CoursePlanner.Repo.Migrations.CreateCourse do
+  use Ecto.Migration
+
+  def change do
+    create table(:courses) do
+      add :name, :string
+      add :description, :string
+      add :number_of_sessions, :integer
+      add :session_duration, :time
+      add :syllabus, :string
+      add :status, :entity_status
```

Comment:
I was thinking since we have updated_at and delete status it works the same way. but if i can drop delete status and add "deleted_at".

However I have doubt now in general of to have soft delete in any case.